### ISSUE TITLE
Filter tx type on treasury page

### ISSUE THREAD

**Issue by martonp** [decred/dcrdata#1924]
I don't see any way to see the list of treasury spends. This would be possible if there was a option to filter by tx type on the treasury page.

**Comment by martonp** [comment]
I would like to work on this if this is a desirable feature.

**Comment by davecgh** [comment]
There is a transaction type selection dropdown at the bottom of the page.  Isn't that what you're referring to?

https://explorer.dcrdata.org/treasury?txntype=tspend

**Comment by davecgh** [comment]
I did notice that the pagination seems to have a few bugs though.  For example, if you refresh the page with a transaction type selected (spend in this case), it'll show the correct number of transaction (e.g. `showing 1 -- 4 of 4 transactions`), but if you select `All` from the type dropdown and then go back to `Spend`, it doesn't update properly (e.g. `showing 1 – 20 of 123,607 transactions).

I also noticed the next link is not disabled when when there are no more results as would be expected.
